DANS-KNAW/easy-deposit-api
# EASY-1820 renamed licenseAgreement to depositAgreement (for JSon only yet)

Fixes EASY-1820 Change acceptLicenseAgreement to acceptDepositAgreement 

#### When applied it will
* have licenseAgreement renamed to depositAgreement in scala objects that are (de)serialized to json
* mention explicit versions of XSD so nothing will break when EASY-SCHEMA upgrades any
* validate the XML with schema's for files.xml and agreements.xml

#### Where should the reviewer @DANS-KNAW/easy start?

#### How should this be manually tested?

* build and deploy together with ui
  * temporarily increment https://github.com/DANS-KNAW/easy-deposit-api/blob/8d3fa55954a21a7f29fcc170959413fea5f718b5/pom.xml#L28
  * as long as tests fail: `mvn clean install -DskipTests=true`
* [ ] check that the service runs: `sudo service easy-deposit-api status`
* go to `http://deasy.dans.knaw.nl/deposit` with a browser, login, create a deposit
* check the box at the bottom of the form to accept the depositor agreement
* save a draft

#### Related pull requests on github
* [ ] fix #89 , it does not start when deployed on http://develop.dans.knaw.nl/boxes/deasy-2018-11-30.box with  PR 267 merged into dtap
* must be merged/deployed together with https://github.com/DANS-KNAW/easy-deposit-ui/pull/113
* Update also [`application.properties`](https://github.com/DANS-KNAW/easy-validate-dans-bag/blob/076bd82cd95d365d2826bf932ba4e946f9cb5b0c/src/main/assembly/dist/cfg/application.properties#L11) of `easy-validate-dans-bag`
